scripts/bayesian_debug_helper.py

```
@@ -0,0 +1,376 @@
+        print("\n" + "-" * 80)
+        print("🔬 ДЕТАЛЬНАЯ ДИАГНОСТИКА")
+        print("-" * 80)
+
+        for i, diagnosis in enumerate(analysis.get("diagnoses", []), 1):
+            print(f"\n{i}. Тест: {diagnosis['test_name']}")
+            print(f"   Тип ошибки: {diagnosis['error_type']}")
+            print(f"   Наиболее вероятная причина: {diagnosis['most_likely_cause']}")
+            print(f"   Вероятность: {diagnosis['probability']:.2%}")
+            print(f"   Уверенность: {diagnosis['confidence']:.2%}")
+
+            print(f"   📋 Доказательства:")
```

Flake8 (7.3.0)

[error] f-string is missing placeholders

(F541)